### PR TITLE
ADF-120: Proto Header

### PR DESCRIPTION
**Pull request by payekumi**
**Jira Ticket**
- [ADF-120: Prototype site header bar](https://palantir.atlassian.net/browse/ADF-120)

## Description
Created an Organism in the style guide called Main navigation Dr. Finder.

## To Test
Checkout the ADF-120-proto-header branch 
cd into style guide 
run gulp serve.
Access the local site with the provided URL
Navigate to Organisms and Main navigation finder (reference image below

Screen Shot 2022-09-29 at 2 56 30 PM

## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

Completed

Screen Shot 2022-09-29 at 3 03 06 PM

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/js/main-navigation-dr-finder.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
+(function ($, Drupal) {
```
Comment (nstriedinger) [inline styleguide/source/assets/js/main-navigation-dr-finder.js]: Do we need all this for simple drop-downs?

File: styleguide/source/assets/scss/02-molecules/_sign-in-dropdown-dr-finder.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
+.dr-finder__sign-in-dropdown {
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/02-molecules/_sign-in-dropdown-dr-finder.scss]: @payekumi Do we need all this?

**Comment by nstriedinger** [comment]
@payekumi Create a markdown file (`.md`) and see other comments.

File: styleguide/source/assets/js/main-navigation-dr-finder.js (hunk 1 above)
Comment (payekumi) [reply styleguide/source/assets/js/main-navigation-dr-finder.js]: This main-navigation-dr-finder.js has been removed

File: styleguide/source/assets/scss/02-molecules/_sign-in-dropdown-dr-finder.scss (hunk 2 above)
Comment (payekumi) [reply styleguide/source/assets/scss/02-molecules/_sign-in-dropdown-dr-finder.scss]: This sign-in-dropdown-dr-finder.scss file has also been removed

**Comment by nstriedinger** [comment]
@payekumi I don't see any `options` here --> http://localhost:3000/?p=molecules-explore-menu-dr-finder

Can we just use the original `@molecules/explore-menu.twig`?

File: styleguide/source/_patterns/03-organisms/dr-finder-main-navigation/dr-finder-main-navigation.twig [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div id="amaCategoryNavigation" class="ama__main-navigation ama__main-navigation--dr-finder">
+  <div class="container">
+    {% include '@molecules/explore-menu-dr-finder.twig' with { exploreMenu : drFinderMainNavigation.exploreMenu } %}
```
Comment (nstriedinger) [inline styleguide/source/_patterns/03-organisms/dr-finder-main-navigation/dr-finder-main-navigation.twig]: maybe use `@molecules/explore-menu.twig`

**Pull request merged**